app/konnect/mesh-manager/federate-zone.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Federate a zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+If you already have a zone control plane which is not connected to any global control plane, you can federate it to {{site.konnect_short_name}} in Mesh Manager. [Mesh Manager](/konnect/mesh-manager/) allows you to create, manage, and view your {{site.mesh_product_name}} service meshes using the {{site.konnect_short_name}} platform.
```

Comment:
@jakubdyszkiewicz For the "you can federate it to", does this mean federated authentication? So using federated auth to connect Konnect and Mesh?